frontend/app/controllers/project/show/transportation.js

```diff
@@ -0,0 +1,7 @@
+import Controller from '@ember/controller';
+import { alias } from '@ember-decorators/object/computed';
+
+export default class ProjectShowTransportationController extends Controller {
+  @alias('model.project') project;
```

Comment:
Is this necessary? It's not used anywhere in the template AFAIK.

Reply:
It is passed into the ExistingConditions component in projects/show/transportation route